plugins/woocommerce/tests/e2e-pw/tests/merchant/create-coupon.spec.js

```diff
@@ -1,52 +1,129 @@
-const { test, expect } = require( '@playwright/test' );
-const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
+const { test: baseTest, expect } = require( '../../fixtures' );
 
-const couponCode = `code-${ new Date().getTime().toString() }`;
+baseTest.describe( 'Coupon management', () => {
+	const couponData = {
+		fixedCart: {
+			code: `fixedCart-${ Date.now() }`,
+			description: 'Simple fixed cart discount',
+			amount: '25',
+		},
+		fixedProduct: {
+			code: `fixedProduct-${ Date.now() }`,
+			description: 'Simple fixed product discount',
+			amount: '5',
+		},
+		percentage: {
+			code: `percentage-${ Date.now() }`,
+			description: 'Simple percentage discount',
+			amount: '10',
+		},
+		expiryDate: {
+			code: `expiryDate-${ Date.now() }`,
+			description: 'Simple expiry date discount',
+			amount: '15',
+			expiryDate: '2023-12-31',
+		},
+		freeShipping: {
+			code: `freeShipping-${ Date.now() }`,
+			description: 'Simple free shipping discount',
+			amount: '1',
+			freeShipping: true,
+		},
+	};
```

Comment:
Refactored the original simple test to use a fixture -- and I like the clarity of this setup!